package.json

```diff
@@ -0,0 +1,61 @@
+{
+    "name": "teamup-client",
+    "version": "1.0.0",
+    "license": "GPL-3.0",
+    "scripts": {
+        "prepare": "husky install",
+        "prebuild": "rimraf build/",
+        "build": "react-scripts build",
+        "start": "react-scripts start",
+        "test": "react-scripts test --passWithNoTests",
+        "format": "prettier --ignore-unknown --ignore-path .gitignore --write .",
+        "lint": "eslint --ext .js,.ts src/"
+    },
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/coders-camp-2021-best-team/TeamUp-Client.git"
+    },
+    "dependencies": {
+        "@emotion/react": "^11.8.2",
+        "@emotion/styled": "^11.8.1",
+        "@mui/icons-material": "^5.5.1",
+        "@mui/material": "^5.5.2",
+        "@testing-library/jest-dom": "^5.14.1",
+        "@testing-library/react": "^12.0.0",
+        "@testing-library/user-event": "^13.2.1",
+        "@types/jest": "^27.0.1",
+        "@types/node": "^16.7.13",
+        "@types/react": "^17.0.20",
+        "@types/react-dom": "^17.0.9",
+        "formik": "^2.2.9",
```

Comment:
Formik? Why not react-hook-forms?

The major difference is that, the formik will rerender out component on each input type, the react-hook-form wouldn't do that

also the react-hook-form has less dependencies which is, obviously, better